cmd/heighliner/github_repository.go

```diff
@@ -20,10 +19,11 @@ var (
 	}
 
 	ghpcFlags struct {
-		Namespace    string `long:"namespace" env:"NAMESPACE" description:"The namespace we'll watch for CRDs. By default we'll watch all namespaces."`
-		Domain       string `long:"domain" env:"DOMAIN" description:"The domain name used for callbacks" required:"true"`
-		InsecureSSL  bool   `long:"insecure-ssl" env:"INSECURE_SSL" description:"Allow insecure callbacks to the webhook"`
-		CallbackPort string `long:"callback-port" env:"CALLBACK_PORT" description:"The port to run the callbacks server on" default:":8080"`
+		Namespace                   string `long:"namespace" env:"NAMESPACE" description:"The namespace we'll watch for CRDs. By default we'll watch all namespaces."`
+		Domain                      string `long:"domain" env:"DOMAIN" description:"The domain name used for callbacks" required:"true"`
+		InsecureSSL                 bool   `long:"insecure-ssl" env:"INSECURE_SSL" description:"Allow insecure callbacks to the webhook"`
+		CallbackPort                string `long:"callback-port" env:"CALLBACK_PORT" description:"The port to run the callbacks server on" default:":8080"`
+		ReconciliationPeriodSeconds int    `long:"reconciliation-period-seconds" env:"RECONCILIATION_PERIOD_SECONDS" description:"How often the controller should check for Github changes missed by webhooks"`
```

Comment:
Should we set a default of say, 5 minutes?

It'd also be cool of people could use the `5m` or `10s` etc. notation (so making it a string and parse it with `time.ParseDuration`.

Reply:
Done. I put the default for 10min, but let me know.